comps/comps-foreman-rhel7.xml

```diff
@@ -34,6 +34,7 @@
       <packagereq type="default">foreman-service</packagereq>
       <packagereq type="default">foreman-telemetry</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-compute-scroll-into-view</packagereq>
```

Comment:
this should be added to `comps-katello-server-rhel7.xml` as `tfm-nodejs-compute-scroll-into-view` and to `comps-katello-el8.xml` as is and removed from here

Reply:
this comment is still outstanding

Reply:
was still updating! It should be good now